src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs

```diff
@@ -13292,6 +13292,96 @@ .maxstack  4
 ");
         }
 
+        [WorkItem(24348, "https://github.com/dotnet/roslyn/issues/24348")]
+        [Fact]
+        public void VarargBridgeSource()
+        {
+
+            var code = @"
+public class VarArgs
+{
+    public void Invoke(__arglist)
```

Comment:
It would need to be in source. How can it differ in modifiers? Example?

---
In reply to: [167042260](https://github.com/dotnet/roslyn/pull/24679#discussion_r167042260) [](ancestors = 167042260)

Reply:
Will spend some time investigating this scenario.

---
In reply to: [167087717](https://github.com/dotnet/roslyn/pull/24679#discussion_r167087717) [](ancestors = 167087717,167042260)